javascript/ql/src/semmle/javascript/security/dataflow/IndirectCommandInjectionCustomizations.qll

```diff
@@ -39,6 +39,22 @@ module IndirectCommandInjection {
     }
   }
 
+  /**
+   * An object containing command-line arguments that were parsed with a default configuration, considered as a flow source for command injection.
```

Comment:
I don't understand what "default configuration" means in this context.

Reply:
I have removed the "default" part from the name and docstring. The important thing is that it has been parsed.